.github/workflows/test.yml

```diff
@@ -15,4 +15,5 @@ jobs:
     steps:
     - uses: actions/checkout@master
     - uses: ./
-    - run: git ls-remote --tags origin
+      with:
```

Comment:
We had a flow to test the v1.2 feature, so we need to update our tests on the revert